tests/tck/features/bugfix/TestYieldConstantAfterPipe.feature

```diff
@@ -15,25 +15,13 @@ Feature: Test yield constant after pipe
     Then the result should be, in any order:
       | a |
       | 1 |
-      | 1 |
-      | 1 |
-      | 1 |
-      | 1 |
-      | 1 |
-      | 1 |
     When executing query:
       """
       GO FROM "Tim Duncan" OVER * | YIELD 1 AS a WHERE true;
       """
     Then the result should be, in any order:
       | a |
       | 1 |
```

Comment:
I want to know what's the right result should be ? @CPWstatic 

Should the number of results be same as the edges from `Tim Duncan` ?

Reply:
According to the origin code, I guess it may want to set input to the inner var when the all expressions of yield columns are constant. So I change it to this.

Reply:
I think it's not compatible with the definition of pipe